rules/windows/process_creation/proc_creation_win_susp_ping_del.yml

```diff
@@ -0,0 +1,28 @@
+title: Suspicious Ping And Del Combination 
+id: 54786ddc-5b8a-11ed-9b6a-0242ac120002
+status: experimental
+description: Detects a method often used by ransomware. Which combines the "ping" to wait a couple of seconds and then "del" to delete the file in question. Its used to hide the file responsible for the initial infection for example
+references:
+    - https://blog.sygnia.co/kaseya-ransomware-supply-chain-attack
+    - https://media.kasperskycontenthub.com/wp-content/uploads/sites/43/2022/06/23093553/Common-TTPs-of-the-modern-ransomware_low-res.pdf
+    - https://www.acronis.com/en-us/blog/posts/lockbit-ransomware/
+    - https://symantec-enterprise-blogs.security.com/blogs/threat-intelligence/blackbyte-exbyte-ransomware
+author: Ilya Krestinichev
+date: 2022/11/03
+tags:
+    - attack.defense_evasion
+    - attack.t1070.004
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        CommandLine|contains|all:
+            # Example: "C:\Windows\System32\cmd.exe"  /C ping 127.0.0.7 -n 3 > Nul & fsutil file setZeroData offset=0 length=524288 "C:\Users\User\Desktop\lockbit\lockbit.exe" & Del /f /q "C:\Users\User\Desktop\lockbit\lockbit.exe".
+            - 'ping '
+            - ' -n'
+            - '& del'
+    condition: selection
```

Comment:
```suggestion
detection:
    # Example: "C:\Windows\System32\cmd.exe"  /C ping 127.0.0.7 -n 3 > Nul & fsutil file setZeroData offset=0 length=524288 "C:\Users\User\Desktop\lockbit\lockbit.exe" & Del /f /q "C:\Users\User\Desktop\lockbit\lockbit.exe".
    selection_count:
        CommandLine|contains:
            - ' -n '
            - ' /n '
    selection_nul:
        CommandLine|contains: 'Nul' # Covers "> Nul" and ">Nul "
    selection_del_param:
        - ' /f '
        - ' -f '
        - ' /q '
        - ' -q '
    selection_all:
        CommandLine|contains|all:
            - 'ping' # Covers "ping" and "ping.exe"
            - 'del '
    condition: all of selection_*
```

Reply:
@frack113 check this suggestion and let me know what you think.

Reply:
Seen good for me